res/openvpn-gui-res-zh-hant.rc

```diff
@@ -101,16 +102,16 @@ END
 /* Change Passphrase Dialog */
 ID_DLG_CHGPASS DIALOG 6, 18, 193, 82
 STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER
-CAPTION "OpenVPN - Change Passphrase"
+CAPTION "OpenVPN - 修改密碼"
```

Comment:
read though. but considering context in Chinese, they are same for "changing" a simple item like password.